go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/MaterializeInc/terraform-provider-materialize
 
-go 1.22.0
+go 1.22.7
```

Comment:
just double checking, is this kind of upgrade a breaking change? /compatible with how we're changing the version of the terraform provider?

Reply:
Yep, this minor version upgrade should be ok, it was needed because of a recent dependabot commit:

https://github.com/MaterializeInc/terraform-provider-materialize/commit/4f61ea1fe8673fe553fa109bab7867802f886ebd

Kind of sneaking the bump into this PR 😅